drivers/spi/spi_ll_stm32.c

```diff
@@ -764,6 +798,10 @@ static int transceive(const struct device *dev,
 
 	spi_context_lock(&data->ctx, asynchronous, cb, userdata, config);
 
+#ifdef CONFIG_PM
```

Comment:
Can we put the #ifdef inside `spi_stm32_pm_policy_state_lock_foo` and remove the #ifdef here to enhance readability ?

Reply:
Done